src/order/bounded_lattice.lean

```diff
@@ -740,6 +740,48 @@ lemma lt_iff_exists_coe_btwn [partial_order α] [densely_ordered α] [no_top_ord
 
 end with_top
 
+namespace subtype
+
+/-- A subtype forms a `⊔`-semilattice if `⊔` preserves the property. -/
+protected def semilattice_sup [semilattice_sup α] {P : α → Prop}
+  (Psup : ∀{{x y}}, P x → P y → P (x ⊔ y)) : semilattice_sup {x : α // P x} :=
```

Comment:
In VScode, typing `\{{` will give you a fancy unicode version of `{{`.

Reply:
```suggestion
  (Psup : ∀⦃x y⦄, P x → P y → P (x ⊔ y)) : semilattice_sup {x : α // P x} :=
```
Typing `\{{}}` (and hitting tab) will give you `⦃⦄` and also put your cursor between them too.

Reply:
Note there are other occurrences, you need to search for `{{`

Reply:
Done (I stopped using the unicode because I was still used to VSCode not substituting these braces correctly.)